test/distributed/org/apache/cassandra/fuzz/topology/AccordTopologyMixupTest.java

```diff
@@ -203,15 +201,15 @@ public String keyspaceName()
         }
     }
 
-    private static class AccordState extends State<Spec> implements SharedState.Listener
+    private static class AccordState extends State<Spec>
     {
-        private final List<Runnable> onError = new CopyOnWriteArrayList<>();
+        private final ListenerHolder listener;
```

Comment:
to enable `HarryOnAccord` without a ton of copy paste, refactor so the logic can be shared